packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-get-att.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: SomeBucketName
+      ReplicationConfiguration:
+        Role: "some-role"
+        Rules:
+        - Destination:
+            Bucket: Bucket1
+            StorageClass: STANDARD
+          Status: Enabled
+  Bucket2:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: 
+        !GetAtt [ Bucket1, BucketName ]
+  Bucket3:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !GetAtt Bucket1.BucketName
+  Bucket4:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !GetAtt Bucket1.Rules.Destination
+  Bucket5:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: 
+        !GetAtt [ Bucket1, Rules.Destination ]
```

Comment:
Again - are we 100% sure this is legal CFN?

Reply:
In addition, the docs show this being JSON:
    `{ "Fn::GetAtt" : [ "logicalNameOfResource", "attributeName" ] }`

and this being YAML:
    `!GetAtt logicalNameOfResource.attributeName`

so the quotes are omitted in both the short and long form syntax. I'll check this later, to make sure the docs aren't lying.

Reply:
I'm asking specifically about `Rules.Destination`. Which is different than the example you have (`!GetAtt logicalNameOfResource.attributeName` - I don't believe `attributeName` can contain `.`).

Reply:
It depends on the attribute name; for example, here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb.html#aws-properties-ec2-elb-return-values

there are attribute names containing `.`